examples/package.json

```diff
@@ -60,21 +60,22 @@
     "test:status-message-serialize-err": "ava __tests__/test-status-message-serialize-err.ava.js",
     "test:status-message-deserialize-err": "ava __tests__/test-status-message-deserialize-err.ava.js",
     "test:status-deserialize-class": "ava __tests__/test-status-deserialize-class.ava.js",
-    "test:basic-updates": "ava __tests__/test-basic-updates.ava.js"
+    "test:basic-updates": "ava __tests__/test-basic-updates.ava.js",
+    "calculate-file-sizes": "WORKING_DIR=$(pwd) pnpm --filter shared-scripts run calculate-file-sizes"
   },
   "author": "Near Inc <hello@nearprotocol.com>",
   "license": "Apache-2.0",
   "dependencies": {
+    "borsh": "^v1.0.0",
```

Comment:
The version specifier `^v1.0.0` is not valid because the `v` is not part of the standard versioning scheme that npm expects.

Reply:
Hmm, I haven't changed anything regarding this, I will check it out.